src/meta/meta_backup_service.cpp

```diff
@@ -1540,15 +1515,28 @@ void backup_service::modify_backup_policy(configuration_modify_backup_policy_rpc
     }
 
     if (request.__isset.start_time) {
+        if (context_ptr->has_backup_history() || context_ptr->is_under_backuping()) {
```

Comment:
Adding `context_ptr->has_backup_history()` condition, it means if the policy has already generated the backup, its start time can not be modified. Could you please explain why it should not be modified to me?

Reply:
Because `start_time` only indicates the first time to start backup, if we have already started a backup, we shouldn't modify it.

Reply:
For once backup, start_time indicates the first time to start backup, for periodical backup, it should be allowed to be updated, for each time, it indicates this round starting backup.

Reply:
The start time of subsequent backups is only related to `last_backup_start_time` and `backup_interval_secends`, if users want to modify next backup time, they could modify `backup_interval_secends`.

Reply:
Yes, you almost persuade me. Let me take an example to explain, there were a policy whose start time is 1:00, interval time is 1 day, if we would like update it to backup everyday 2:00. In this case, we wouldn't update interval time, if we don't provide updating start_time, we can only create another policy or update interval twice.

Reply:
Let‘s imagine how would user use this feature, maybe it's almost like how would you set you clock, there is no such a configurable "interval" concept, the interval should always be fixed value like day(or week, month, etc).
If user want to backup multiple times in one day, she should set multiple policies. On the other hand, I believe backup multiple times a day isn't a regular requirement, we don't need to make our design and code too complex to satisfy it.

Reply:
I don't think the design and code is complex. `start_time` is the first time to start a backup of a policy, `backup_interval_seconds` is interval between two backups, which is quite easy to understand. If users want to modify start_time, they should create another policy.

If we all agree that we should keep `backup_interval_seconds`, then any interval could be set, we shouldn't only support the fixed interval(day/week/month).